schema.sql

```diff
@@ -8,3 +8,7 @@ CREATE TABLE animals (
   neutered BOOLEAN,
   weight_kg DECIMAL
 );
+
+--ADD NEW COLUMN
+ALTER TABLE animals
+ADD COLUMN species VARCHAR;
```

Comment:
- [Optional] Hi @HossainAraf, I think it is important to validate the number of input characters for each column: varchar(255)